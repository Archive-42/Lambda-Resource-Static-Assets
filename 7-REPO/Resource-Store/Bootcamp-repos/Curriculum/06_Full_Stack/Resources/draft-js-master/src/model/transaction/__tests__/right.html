<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>__tests__</title>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/bgoonz/GIT-CDN-FILES/right.css">
</head>

<body>

<table border="0" cellspacing="1" cellpadding="2" summary="file table" class="file-table centered">
<tr><td colspan="3" class="header-row"><b>Folders</b></td></tr>
<tr><td style="text-align: center"><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/../right.html" target="rframe">&lt;parent&gt;</a></td><td style="text-align: center">&nbsp;<a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/right.html" target="rframe">__snapshots__</a></td></tr>
</table>
<br>

<table border="0" cellspacing="1" cellpadding="2" summary="file table" class="file-table centered">
<tr><th><b>File</b></th><th><b>Size</b></th><th><b>File</b></th><th><b>Size</b></th><th><b>File</b></th><th><b>Size</b></th></tr>
<tr><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/right.html" >right.html</a></td><td align="right">5,372</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/insertFragmentIntoContentState-test.js" >insertFr...st.js</a></td><td align="right">9,268</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/randomizeBlockMapKeys-test.js" >randomiz...st.js</a></td><td align="right">3,054</td></tr>
<tr class="row2"><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/applyEntityToContentBlock-test.js" >applyEnt...st.js</a></td><td align="right">1,105</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/insertIntoList-test.js" >insertIn...st.js</a></td><td align="right">1,015</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/removeEntitiesAtEdges-test.js" >removeEn...st.js</a></td><td align="right">4,534</td></tr>
<tr><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/applyEntityToContentState-test.js" >applyEnt...st.js</a></td><td align="right">1,820</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/insertTextIntoContentState-test.js" >insertTe...st.js</a></td><td align="right">2,068</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/removeRangeFromContentState-test.js" >removeRa...st.js</a></td><td align="right">7,262</td></tr>
<tr class="row2"><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/ContentStateInlineStyle-test.js" >ContentS...st.js</a></td><td align="right">2,569</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/moveBlockInContentState-test.js" >moveBloc...st.js</a></td><td align="right">4,087</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/splitBlockInContentState-test.js" >splitBlo...st.js</a></td><td align="right">5,662</td></tr>
<tr><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/getContentStateFragment-test.js" >getConte...st.js</a></td><td align="right">3,799</td><td></td><td></td><td></td><td></td></tr>
</table>

</body>
</html>