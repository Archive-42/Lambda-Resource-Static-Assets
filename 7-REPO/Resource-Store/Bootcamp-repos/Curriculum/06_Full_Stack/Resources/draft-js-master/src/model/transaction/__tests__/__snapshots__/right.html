<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>__snapshots__</title>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/bgoonz/GIT-CDN-FILES/right.css">
</head>

<body>

<table border="0" cellspacing="1" cellpadding="2" summary="file table" class="file-table centered">
<tr><td colspan="3" class="header-row"><b>Folders</b></td></tr>
<tr><td style="text-align: center"><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/../right.html" target="rframe">&lt;parent&gt;</a></td></tr>
</table>
<br>

<table border="0" cellspacing="1" cellpadding="2" summary="file table" class="file-table centered">
<tr><th><b>File</b></th><th><b>Size</b></th><th><b>File</b></th><th><b>Size</b></th><th><b>File</b></th><th><b>Size</b></th></tr>
<tr><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/right.html" >right.html</a></td><td align="right">5,425</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/insertFragmentIntoContentState-test.js.snap" >insertFr....snap</a></td><td align="right">26,661</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/randomizeBlockMapKeys-test.js.snap" >randomiz....snap</a></td><td align="right">6,327</td></tr>
<tr class="row2"><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/applyEntityToContentBlock-test.js.snap" >applyEnt....snap</a></td><td align="right">2,212</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/insertIntoList-test.js.snap" >insertIn....snap</a></td><td align="right">427</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/removeEntitiesAtEdges-test.js.snap" >removeEn....snap</a></td><td align="right">40,280</td></tr>
<tr><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/applyEntityToContentState-test.js.snap" >applyEnt....snap</a></td><td align="right">12,369</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/insertTextIntoContentState-test.js.snap" >insertTe....snap</a></td><td align="right">13,000</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/removeRangeFromContentState-test.js.snap" >removeRa....snap</a></td><td align="right">49,658</td></tr>
<tr class="row2"><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/ContentStateInlineStyle-test.js.snap" >ContentS....snap</a></td><td align="right">25,605</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/moveBlockInContentState-test.js.snap" >moveBloc....snap</a></td><td align="right">6,825</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/splitBlockInContentState-test.js.snap" >splitBlo....snap</a></td><td align="right">42,138</td></tr>
<tr><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/7-REPO/Resource-Store/Bootcamp-repos/Curriculum/06_Full_Stack/Resources/draft-js-master/src/model/transaction/__tests__/__snapshots__/getContentStateFragment-test.js.snap" >getConte....snap</a></td><td align="right">10,372</td><td></td><td></td><td></td><td></td></tr>
</table>

</body>
</html>